============================= test session starts ==============================
platform linux -- Python 3.7.6, pytest-5.3.5, py-1.8.1, pluggy-0.13.1 -- /home/luisclaudino/anaconda3/bin/python
cachedir: .pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/home/luisclaudino/codenation/python-5/.hypothesis/examples')
rootdir: /home/luisclaudino/codenation/python-5
plugins: doctestplus-0.5.0, arraydiff-0.3, openfiles-0.4.0, remotedata-0.3.2, astropy-header-0.1.2, hypothesis-5.5.4
collecting ... collected 8 items

test_submit.py::TestChallenge1::test_len PASSED                          [ 12%]
test_submit.py::TestChallenge1::test_1 PASSED                            [ 25%]
test_submit.py::TestChallenge1::test_2 PASSED                            [ 37%]
test_submit.py::TestChallenge1::test_3 PASSED                            [ 50%]
test_submit.py::TestChallenge1::test_4 PASSED                            [ 62%]
test_submit.py::TestChallenge1::test_5 PASSED                            [ 75%]
test_submit.py::TestChallenge1::test_6 PASSED                            [ 87%]
test_submit.py::TestChallenge1::test_7 PASSED                            [100%]

=============================== warnings summary ===============================
/home/luisclaudino/anaconda3/lib/python3.7/site-packages/_pytest/junitxml.py:436
  /home/luisclaudino/anaconda3/lib/python3.7/site-packages/_pytest/junitxml.py:436: PytestDeprecationWarning: The 'junit_family' default value will change to 'xunit2' in pytest 6.0.
  Add 'junit_family=xunit1' to your pytest.ini file to keep the current format in future versions of pytest and silence this warning.
    _issue_warning_captured(deprecated.JUNIT_XML_DEFAULT_FAMILY, config.hook, 2)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
---- generated xml file: /home/luisclaudino/codenation/python-5/output.xml -----
========================= 8 passed, 1 warning in 0.22s =========================
